packages/shopify-app-session-storage-prisma/src/prisma.ts

```diff
@@ -128,6 +128,18 @@ export class PrismaSessionStorage<T extends PrismaClient>
       userId:
         (sessionParams.onlineAccessInfo?.associated_user
           .id as unknown as bigint) || null,
+      firstName:
```

Comment:
We could wrap this in an (if session.isOnline) to make this only a breaking change if you are using online tokens.

Reply:
But we then would probably have to make some other changes, to have to handle two possible DB schemas. 🤷